install.php

```diff
@@ -182,6 +182,14 @@
 
 	<table cellspacing="0">
 		<tr>
+			<th>Composer Installed</th>
+			<?php if (is_dir($vendor_path) AND is_file($vendor_path.'autoload.php')): ?>
```

Comment:
Please define `$vendor = '../vendor/';` at the top; it shouldn't use the variable from `index.php`.